jupyter/datascience/ubi8-python-3.8/mongodb-org-6.0.repo-x86_64

```diff
@@ -0,0 +1,6 @@
+[mongodb-org-6.0]
+name=MongoDB Repository
+baseurl=https://repo.mongodb.org/yum/redhat/9/mongodb-org/6.0/x86_64/
```

Comment:
```suggestion
baseurl=https://repo.mongodb.org/yum/redhat/8/mongodb-org/6.0/x86_64/
```

Reply:
Thanks for the review! :slightly_smiling_face:  
Question: why do we need to use from the baseurl on mongo and msql the 8 instead of 9?